### PR TITLE
Delete RK files when we're done with them

### PR DESCRIPTION
**Pull request by Erin-Mounts**
so they don't keep piling up and using up storage space

### CODE REVIEW

File: BridgeAppSDK/SBAScheduledActivityManager.swift [hunk 1]
```diff
@@ -322,8 +322,22 @@ open class SBAScheduledActivityManager: NSObject, SBASharedInfoController, ORKTa
...
+    /**
+     Delete the output directory.
+     */
+    @objc(deleteOutputDirectoryForTaskViewController:)
+    open func deleteOutputDirectory(for taskViewController: ORKTaskViewController) {
+        guard let outputDirectory = taskViewController.outputDirectory else {return}
+        do {
+            try FileManager.default.removeItem(at: outputDirectory)
+        } catch let error as NSError {
+            debugPrint("Unable to remove ResearchKit output directory at \(outputDirectory): \(error)")
```
Comment (syoung-smallwisdom) [inline BridgeAppSDK/SBAScheduledActivityManager.swift]: Won't this mean that in the release build that the error is silenced?

File: BridgeAppSDK/SBAScheduledActivityManager.swift [hunk 2]
```diff
@@ -347,7 +361,38 @@ open class SBAScheduledActivityManager: NSObject, SBASharedInfoController, ORKTa
             taskViewController.task?.resetTrackedDataChanges()
         }
         
-        taskViewController.dismiss(animated: true) {}
+        taskViewController.dismiss(animated: true) {
+            self.offMainQueue.async {
+                self.deleteOutputDirectory(for: taskViewController)
+                #if DEBUG
+                DispatchQueue.main.async {
```
Comment (syoung-smallwisdom) [inline BridgeAppSDK/SBAScheduledActivityManager.swift:369]: why going back on main thread?

File: BridgeAppSDK/SBAScheduledActivityManager.swift (hunk 1 above)
Comment (Erin-Mounts) [reply BridgeAppSDK/SBAScheduledActivityManager.swift]: True. I'm never sure where the right place is to draw the line between "things we might need to troubleshoot in production apps" and "things we want to catch before it goes to production but don't want to clutter the logs with/potentially reveal sensitive info/etc. in production apps".

File: BridgeAppSDK/SBAScheduledActivityManager.swift (hunk 2 above)
Comment (Erin-Mounts) [reply BridgeAppSDK/SBAScheduledActivityManager.swift:369]: I've had issues in the past with logging to the console from random threads leaving some debug output interleaved with other output, making it confusing and hard to follow. Not sure it would be a problem here, or that this would even actually help with that if it were though, so... yeah. You want I should remove it?

Comment (syoung-smallwisdom) [reply BridgeAppSDK/SBAScheduledActivityManager.swift:369]: right, that makes sense. I was just curious. thanks.

File: BridgeAppSDK/SBAScheduledActivityManager.swift (hunk 1 above)
Comment (syoung-smallwisdom) [reply BridgeAppSDK/SBAScheduledActivityManager.swift]: What about splitting into two lines: if debug print the directory, if not then just print the error reason?

**Review by syoung-smallwisdom — APPROVED** [review]
